dataland-frontend/tests/e2e/specs/data-ownership/DataOwnershipUserJourney.ts

```diff
@@ -0,0 +1,85 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { admin_name, admin_pw, reader_name, reader_pw } from "@e2e/utils/Cypress";
+import { getKeycloakToken, login, logout } from "@e2e/utils/Auth";
+import {
+  CompanyDataControllerApi,
+  type CompanyDataOwners,
+  Configuration,
+  DataTypeEnum,
+  type PathwaysToParisData,
+} from "@clients/backend";
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from "@e2e/utils/CompanyUpload";
+import { uploadFrameworkData } from "@e2e/utils/FrameworkUpload";
+import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { ARRAY_OF_FRAMEWORKS_WITH_UPLOAD_FORM } from "@/utils/Constants";
+
+let p2pFixtureForTest: FixtureData<PathwaysToParisData>;
+before(function () {
+  cy.fixture("CompanyInformationWithP2pPreparedFixtures").then(function (jsonContent) {
+    const preparedFixturesP2p = jsonContent as Array<FixtureData<PathwaysToParisData>>;
+    p2pFixtureForTest = getPreparedFixture("P2p-dataset-with-no-null-fields", preparedFixturesP2p);
+  });
+});
+
+describeIf(
+  "As a user, I expect to be able to upload data for one company for which iam data owner",
+  {
+    executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+  },
+  function (): void {
+    beforeEach(() => {
+      cy.ensureLoggedIn(admin_name, admin_pw);
+    });
```

Comment:
beforeEach hook is not necessary here.

There is only one test, so "ensureLoggedIn" can happen there.

Reply:
Changed